packages/design-tokens/docs/pages/how-to-use-with-sass.stories.mdx

```diff
@@ -0,0 +1,57 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import exampleSass from '!!raw-loader!./examples/example.scss'
+import { CodeBlock, LinkToStory } from '../DocsComponents'
+import apiPage from './api.stories.mdx'
+import gettingStartedPage from './how-to-setup.stories.mdx'
+
+<Meta title="How-to guides/How to use design tokens in SASS code" parameters={{ docsOnly: true }} />
+
+# How to use design tokens in SASS code
+
+
+## 1. Setup a ThemeManager
+
+Ensure you have followed the steps in <LinkToStory storyModule={gettingStartedPage} /> to setup your application for using design tokens.
+
+## 2. Start using design tokens
+
+Tokens (which contain CSS variables, e.g `var(--kz-var-color-ash)`) can be subscribed to in SASS by importing variables exposed by the package.
+
+Example:
+
+<CodeBlock language="scss" code={exampleSass} />
+
+See <LinkToStory storyModule={apiPage} hash="sass-variables">SASS Variables</LinkToStory> for all available tokens.
```

Comment:
We should try use the same name for the link and the page title. Is it "API" or "SASS Variables" or something else?